### ISSUE TITLE
Pancreas ontology expansion for colouring of anatomogram shapes/structures

### ISSUE THREAD

**Issue by sfexova** [ebi-gene-expression-group/atlas-web-single-cell#128]
The ontology expansion and so the colouring of parent structures for cell types across images in the pancreas anatomogram stack is a bit patchy:
In none of the experiments below 'pancreatic acinus' structure/shape is active (grey colour) although the data include acinar cells, which is a child term for pancreatic acinus: https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000622
screenshot here:
[screenshot.pptx](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/files/5850010/screenshot.pptx)

https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-GEOD-81547/results/anatomogram
https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5061/results/anatomogram
https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-ENAD-27/results/anatomogram

This one doesn't have either the acinus or the islet of Langerhans in grey, which is especially weird as the expansion from islet cell types to islet (structure) and the colouring of these across the two images works in the other pancreas experiments
https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5061/results/anatomogram

**Comment by pcm32** [comment]
Thanks @sfexova, I suspect that this might be fixed when we move to the newer version of solr cloud and to the new version of the ontology (both things should happen at the same time). This was delayed because the loading of data with this was failing. We keep this open though.

**Comment by sfexova** [comment]
Yes, but I don't think that can explain all of it, really. These are connection present in the 'old' ontology (EFO) already and Alfonso said we're still using that one for this release. Furthermore, the same connection appears to be 'seen' by some experiments but not others:
all of the experiments above share most of the islet of Langerhans cell types (pancreatic A cell, pancreatic D cell, type B pancreatic cell) and yet in here: https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5061/results/anatomogram
not even the islet of Langerhans (the roundish blob next to the branchy structure) is highlighted in the pacreas main picture. And that makes me also think - why the anatomograms can pick up this one connection from this ontology and not the other from the same one (e.g. acinar cell >> pancreatic acinus) - but yes, I gues that could be Solr??

**Comment by alfonsomunozpomer** [comment]
Silvie’s right, this is a problem in the code. We’re not collecting parents and other relationship accessions (`part_of`, `located_in`) to colour the anatomogram, only the field `ontology_annotation`. I’ve created a ticket associated this issue.

**Comment by pcm32** [comment]
Great that you found the source of the error @alfonsomunozpomer.

**Comment by pcm32** [comment]
@sfexova can you check whether this is still an issue against the current wwwdev with the new v5 collections? Thanks

**Comment by sfexova** [comment]
yes - the islet of Langerhans is now highlighted in the main picture as it should be (but I that was just some weird glitch in one of the experiments as it was OK in the others even before but the main issue) I was drawing attention to, with the acini (that were never working as expected in any of the pancreas experiments we have), is still there - see attached pic for details/explanations, example taken from https://wwwdev.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5061/results/anatomogram but the same issue appears in the other pancreas experiments too
[pancreas_ontology_info_propagation_issues.pdf](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/files/6309363/pancreas_ontology_info_propagation_issues.pdf)
